app/javascript/vue/global_search/groups/protocols.vue

```diff
@@ -3,12 +3,56 @@
     <h2 class="flex items-center gap-2 mt-0 mb-4">
       <i class="sn-icon sn-icon-protocols-templates"></i>
       {{ i18n.t('search.index.protocol_templates') }}
+      [{{ total }}]
     </h2>
+    <div>
+      <div class="grid grid-cols-[500px_110px_200px_200px_250px_auto] items-center">
```

Comment:
i think this not work our smallest screen around 800px and this just not fit. I already discussed with Ana that this not good idea. For now just make hardcoded size for `code` and all other auto